UACloudLibraryServer/Components/Pages/Explorer.razor

```diff
@@ -131,7 +131,7 @@
     private async Task DownloadFileFromURL(UANameSpace item)
     {
         var fileName = $"{item.Title}.NodeSet2.xml";
-        var fileURL = $@"/infomodel/downloadNodeSetXml/{item.Nodeset.Identifier}";
+        var fileURL = $@"/infomodel/download/{item.Nodeset.Identifier}?getNodeSetXML=true";
```

Comment:
This parameter name is misleading. Please rename to NodesetXMLOnly.

Reply:
I have changed the parameter name.